debian/octoscreen.service

```diff
@@ -8,6 +8,7 @@ Type=notify
 NotifyAccess=all
 EnvironmentFile=/etc/octoscreen/config
 ExecStart=/usr/bin/xinit /usr/bin/OctoScreen -- :0 -nolisten tcp -nocursor
+ExecStartPost=/bin/bash -xc '/usr/bin/systemctl stop --quiet octosplash && exit 0 || exit 0'
```

Comment:
Stops the `octosplash` service so, if it is animated, it does not run in the background and on top of OctoScreen.  Will always exit successfully, even if the service isn't running.